patches/ethereum-ens++swarm-js+0.1.40.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/ethereum-ens/node_modules/swarm-js/package.json b/node_modules/ethereum-ens/node_modules/swarm-js/package.json
```

Comment:
Patching doesn't work since we need to install the deps, patch their package.json then reinstall their deps again, which npm & patch-package  don't support.